src/test/java/marquez/client/models/JsonGenerator.java

```diff
@@ -121,6 +121,11 @@ private static String newJsonFor(final DbTable dbTable) {
         .put("createdAt", ISO_INSTANT.format(dbTable.getCreatedAt()))
         .put("updatedAt", ISO_INSTANT.format(dbTable.getUpdatedAt()))
         .put("sourceName", dbTable.getSourceName())
+        .put(
+            "lastModified",
+            dbTable.getLastModified() != null
```

Comment:
By wrapping `Dataset.lastModified` in an `Optional`, we can simplify the `null` check to:

```java
dbTable. getLastModified().map(ISO_INSTANT::format).orElse(null)
```

Reply:
Done